### ISSUE TITLE
Users would like to be able to copy processing and analysis workflows + studies

### ISSUE THREAD

**Issue by adswafford** [qiita-spots/qiita#2208]
Users would like to be able to save processing and analysis workflows to their profile so they can uniformly process their data across studies and analyses

**Comment by josenavas** [comment]
After discussion it also makes sense to have the "default workflows".

**Comment by wasade** [comment]
From chatting with @antgonza, it would be great if we could get a default workflow setup for illumina amplicon data where a series of predefined trims are automatically applied. We discovered that there are a series of preps where there is 100nt processing but not 90nt processing, and that discrepancy results from manually processing different trims.

**Comment by antgonza** [comment]
I think this should be worked on after the `labman` all hands on effort. Thus, not gonna be ready for the current release but hopefully for the next one.
cc: @josenavas

**Comment by antgonza** [comment]
Note that this is the larger issue of https://github.com/biocore/qiita/issues/2207, so just moving this section here.

I can't recall what the original request we got was to do this, but I can think of two use cases:
1) There is a public study that I want to work with, but I want to change/collapse/group/etc. the metadata in a different way for an analysis; Rather than downloading the metadata, prep, fastq etc. and uploading everything fresh and processing again, I want to just copy the study and edit the metadata; Maybe this can be handled better by allowing users to modify and upload the metadata associated with an analysis?
2) I've tried to process my files and completely messed something up and I can't get everything deleted because there are blocking files somewhere and/or I'm confused. I don't want to have to download and upload all the files again, so I just want to duplicate a study and then delete down to the point where I can restart. Although in this case, you'd want the study to just copy the files (metadata, preps, and raw data) w/o associating them I guess?
